src/components/Screen.js

```diff
@@ -1,21 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-/* eslint-disable react/destructuring-assignment */
+
 class Screen extends React.Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
 
   render() {
-    const total = this.props.display.total ? this.props.display.total : '0';
+    const { display } = this.props;
```

Comment:
[OPTIONAL] You can even improve the destructuring to next level, this will make your code look even cleaner! 😅 

```
const { display: { total, operation, next } } = this.props;
```

Reply:
Thanks for the feedback! Change implemented before merging.